l2env/l2env.go

```diff
@@ -1,43 +1,39 @@
-package env
+package l2envpackege
```

Comment:
`l2env` should be fine. Spelling error as well.

Reply:
Yes